openlibrary/templates/books/edit/edition.html

```diff
@@ -379,7 +379,7 @@
                     <label for="is-translation">$_("No")</label>
                 </div>
                 <div class="input">
-                    <input type="radio" name="edition--translation" id="is-translation2" value="yes" $checked_yes onclick="\$('.transYes').customFadeIn();"/>
+                    <input type="radio" name="edition--translation" id="is-translation2" value="yes" $checked_yes onclick="\$('.transYes').removeClass('hidden').customFadeIn();"/>
```

Comment:
Should the `hidden` class be added back when the radio button is unchecked?

Reply:
Not necessary as the JS takes care of that with the call to customFadeIn adding a display: none